src/auto/constants.rs

```diff
@@ -514,6 +514,13 @@ pub static FILE_ATTRIBUTE_UNIX_UID: once_cell::sync::Lazy<&'static str> =
             .to_str()
             .unwrap()
     });
+#[cfg(any(feature = "v2_64", feature = "dox"))]
+pub static MEMORY_MONITOR_EXTENSION_POINT_NAME: once_cell::sync::Lazy<&'static str> =
```

Comment:
You could enable the memory monitor API now, that should all be autogeneratable

Reply:
Added `MemoryMonitor`